lib/cc2650.js

```diff
@@ -103,9 +104,7 @@ CC2650SensorTag.prototype.setMPU9250Period = function(period, callback) {
 
 CC2650SensorTag.prototype.enableMPU9250 = function(mask, callback) {
   this.mpu9250mask |= mask;
-
-  // for now, always write 0x007f, magnetometer does not seem to notify is specific mask is used
-  this.writeUInt16LECharacteristic(MPU9250_UUID, MPU9250_CONFIG_UUID, 0x007f, callback);
+  this.writeUInt16LECharacteristic(MPU9250_UUID, MPU9250_CONFIG_UUID, this.mpu9250mask, callback);
```

Comment:
This part causes issues, and was reverted to resolve #64.

Are there any advantages or disadvantages to always enabling WOM?